roofit/roofitcore/src/RooFuncWrapper.cxx

```diff
@@ -55,6 +57,8 @@ RooFuncWrapper::RooFuncWrapper(const char *name, const char *title, RooAbsReal &
 
    func = buildCode(obj);
 
+   declareToInterpreter("#pragma cling optimize(2)");
```

Comment:
We should turn that into `optimize(0)` somewhere once we are done. Maybe add a RAII object of some sort?

Reply:
Thanks, good idea!

Reply:
I'll do this in another PR, as I get errors like this when I try your suggestion:
```txt
cling::PHOptLevel: conflicting `#pragma cling optimize` directives: was already set to 2
Setting to lower value of 0
```
Since the pragma code was only moved around in this PR without changes, I think it's fair to merge this PR as is because it doesn't relate to the optimization levels